src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -155,7 +170,23 @@ impl CatalogState {
                 Datum::from(az),
             ]),
             diff,
+        }];
+
+        if let (Some(processes), Some(cpu), Some(MemoryLimit(ByteSize(memory)))) =
+            (processes, cpu, memory)
+        {
+            updates.push(BuiltinTableUpdate {
+                id: self.resolve_builtin_table(&MZ_CLUSTER_REPLICA_SIZES),
+                row: Row::pack_slice(&[
+                    Datum::UInt64(id),
+                    Datum::UInt64(u64::cast_from(processes.get())),
+                    Datum::UInt64(u64::cast_from(cpu.as_millicpus()) * 1_000_000),
```

Comment:
Is it possible for this to ever overflow?

Reply:
No, it will generally be less than 100 billion.